src/MoBi.Core/OSPSuite.Core/Domain/Builder/ExpressionTypes.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace OSPSuite.Core.Domain.Builder
+{
+   public static class ExpressionTypes
+   {
+      public static ExpressionType TransportProtein = new ExpressionType(_transporter);
```

Comment:
yes sometning like this. We can build upon after with display name etc.. (probably a real enum to serialize BTW)